packages/neutrino/bin/start.js

```diff
@@ -7,11 +7,7 @@ module.exports = (middleware, options) => {
   return start(middleware, options)
     .fork((errors) => {
       spinner.fail('Building project failed');
-      errors.forEach((err) => {
-        console.error(err.stack || err);
-        err.details && console.error(err.details);
-      });
-
+      errors.forEach(err => console.error(err));
```

Comment:
Problem is, forEach passes currentValue, index, array... We only want to log currentValue.